canvas_modules/common-canvas/__tests__/common-properties/components/title-editor-test.js

```diff
@@ -94,172 +109,184 @@ describe("title-editor renders correctly", () => {
 				help={help}
 			/>
 		);
-		const helpButton = wrapper.find(".properties-title-editor-btn[data-id='help']").hostNodes();
-		expect(helpButton).to.have.length(1);
-		helpButton.simulate("click");
+		const { container } = wrapper;
+		const helpButtonCont = container.querySelector(".properties-title-editor-btn[data-id='help']");
+		expect(helpButtonCont).to.exist;
+		fireEvent.click(helpButtonCont);
 	});
 	it("test with no help", () => {
-		const wrapper = mountWithIntl(
+		const wrapper = renderWithIntl(
 			<TitleEditor
 				store={controller.getStore()}
 				controller={controller}
 				helpClickHandler={helpClickHandler}
 				labelEditable
 			/>
 		);
-		expect(wrapper.find(".properties-title-editor-btn[data-id='help']")).to.have.length(0);
+		const { container } = wrapper;
+		expect(container.querySelector(".properties-title-editor-btn[data-id='help']")).to.not.exist;
 	});
 	it("test help button without a callback", () => {
 		helpClickHandler.resetHistory();
-		const wrapper = mountWithIntl(
+		const wrapper = renderWithIntl(
 			<TitleEditor
 				store={controller.getStore()}
 				controller={controller}
 				labelEditable
 				help={help}
 			/>
 		);
-		const helpButton = wrapper.find(".properties-title-editor-btn[data-id='help']").hostNodes();
-		helpButton.simulate("click");
+		const { container } = wrapper;
+		const helpButton = container.querySelector(".properties-title-editor-btn[data-id='help']");
+		fireEvent.click(helpButton);
 	});
 	it("test edit link", () => {
 		helpClickHandler.resetHistory();
-		const wrapper = mountWithIntl(
+		const wrapper = renderWithIntl(
 			<TitleEditor
 				store={controller.getStore()}
 				controller={controller}
 				labelEditable
 				help={help}
 			/>, { attachTo: document.body }
 		);
-		const titleEdit = wrapper.find(".properties-title-editor-btn[data-id='edit']").hostNodes();
-		titleEdit.simulate("click");
-		expect(wrapper.find("input").is(":focus")).to.be.true;
+		const { container } = wrapper;
+		const input = container.querySelector("input");
+		const titleEdit = wrapper.getAllByRole("button")[0];
```

Comment:
can we retrieve this using same query as `properties-title-editor-btn[data-id='edit']` ?

Reply:
I changed it